chain/cache.go

```diff
@@ -20,6 +20,7 @@ func newCache(maxSize int) *cache {
 
 func (c *cache) GetOrLoad(key interface{}, load func() (interface{}, error)) (interface{}, error) {
 	if value, ok := c.Get(key); ok {
+		metricBlockRepositoryCounter().AddWithLabel(1, map[string]string{"type": "read", "target": "cache"})
```

Comment:
These metrics are per object, as described in the title. I do not know why #910 was done since cache miss/rate was already covered [by this](https://github.com/vechain/thor/blob/9d359f702b32712cf8acd4a4fc53393419509c35/muxdb/internal/trie/metrics.go#L9).

But it is true that [this other cache](https://github.com/vechain/thor/blob/9d359f702b32712cf8acd4a4fc53393419509c35/chain/metric.go#L9) looks like a duplicate since it is placed precisely in receipts, transactions and so on with the difference that only applies to reads.

I can align both things. Wdyt?

Reply:
I would have put this metric here, but I also wanted to record the domain cache/hit, ie `blocks`, `transactions` or `receipts`